chsdi/models/vector/uvek.py

```diff
@@ -604,6 +604,35 @@ class ENERGIEFORSCHUNG(Base, Vector):
 register('ch.bfe.energieforschung', ENERGIEFORSCHUNG)
 
 
+class MinergieGebaeude(Base, Vector):
+    __tablename__ = 'minergiegebaeude'
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/minergiegebaeude.mako'
+    __bodId__ = 'ch.bfe.minergiegebaeude'
+    __extended_info__ = False
```

Comment:
If you don't put the attribute -> means no extended info, so you could remove it.